frontend/src/homePage/LandingPage.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './LandingPage.css';
+import LoginButton from './LoginButton.js';
+import { motion } from "framer-motion";
+
+class LandingPage extends React.Component {
+    constructor(props) {
+      super(props);
+      this.message = "Get Started";
+      this.color = "#5A90A1";
+      this.clickHandling = this.props.loginClick;
+      if (this.props.loginStatus) {
+        this.message = "My Folders";
+        this.color = "#F4B400"
+        this.clickHandling = this.props.folderClick;
+      }
+    }
+  
+    render() {
+      return (
+        <div className="LandingPage">
+          <h1>Frame.cards learning made easy</h1>
+          <motion.div whileHover={{scale:1.2}}>
+            <LoginButton
+              status={this.message}
+              color={this.color}
```

Comment:
as discussed in our 1:1, look for opportunities to separate out styling vs content when possible, if and when it makes sense at least... Not entirely sure how react.js proscribes mixing the two